app/lib/core/models/dialogs.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/foundation.dart';
+
+class DialogRequest {
```

Comment:
```
launchAlertDialog(BuildContext context,
        {String title = "Title", String warning = "Warning"}) =>
    showDialog(
        context: context,
        builder: (context) => AlertDialog(
              title: Text(title),
              content: Text(warning),
            ));
```
As you can see, this accepts a context. But since we want a total separation of UI to business logic, we lose the ability to get the context. Hence to why we use Globalkey instead, which is can be used a sub. for context.

Reply:
Should we remove `launchAlertDialog` now, to prevent people from using it instead of `DialogService`?

Reply:
Will do in my next commit